tools/dev/poetrypkg.py

```diff
@@ -105,8 +105,8 @@ def convert_dependencies(cls, package, dependencies):
 def gen_frozensetup(package_dir, output):
     """Generate a frozen setup suitable for distribution.
     """
-    from poetry.masonry.builders import sdist
-    from poetry.factory import Factory
+    from poetry.core.masonry.builders import sdist
+    from poetry.core.factory import Factory
```

Comment:
we need locked packages for frozensetup which means using the poetry pkg factory impl